### DISCUSSION TITLE
Progress bar in homebrew

### DISCUSSION THREAD

**Discussion by kerhbal** [Homebrew/discussions#465] [Everyday usage]
As I remembered before, the progress bar was something "one-line", something like this:
```
#################40%-------------------------
```
and it will refresh and become 100% after some time.

But now I found the progress bar turns into a multi-line block:
from time to time, there will be another line appended after whenever progress percentage increases.
```
==> Downloading https://homebrew.bintray.com/bottles/python%403.9-3.9.1_4.cat
==> Downloading from https://d29vzk4ow07wi7.cloudfront.net/43cb32e99284961180
0.####
6.################### 
26.#############################      
41.################################################
68.##############################################################
87.######################################################################## 100
```
I think this UI is kind of not so efficient compared to old one, just here to check if anyone agrees to this or knows the reason why it evolves to what looks like right now, thanks

**Comment by SMillerDev** [comment]
This is the progressbar from cURL that homebrew uses. It depends on your shell settings how it displays.

**Reply by kerhbal** [reply]
thanks, I'll dig into it
